abb_irb1200_gazebo/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>abb_irb1200_gazebo</name>
+  <version>0.0.1</version>
+  <description>
+    <p>
+      Gazebo support for the ABB IRB 1200
+    </p>
+    <p>
+      This package contains configuration and launch files
+      to simulate the ABB IRB 1200 with Gazebo.
```

Comment:
Could you please mention the specific variant for which you added Gazebo support?

Reply:
Also: I believe it would be good to document *where* the dynamic properties came from. Especially since these are guestimates (however nice they are), it would be good for future users to understand how you arrived at them to be able to gauge their usability in certain (their) contexts.

Reply:
I added some information to the `package.xml` files. Let me know if that's not enough.